### PR TITLE
Folders for coursera Notes and assignments

### PR DESCRIPTION
**Pull request by LakshayaSinghal**
Added following folders.
1. Linear Algebra Notes(3B1B)
2. Coursera Andrew NG Deep Learning course notes(Course 1, Course 2, Course 4)
3. Coursera Assignments.

### CODE REVIEW

**Comment by Neel-Shah-29** [comment]
@Greyless this is not the way to write the PR Title. Please provide the correct name to PR by editing the title.
Also your commit message is very vague if i merge this commit to master everyone can see this commit message which is not good, so please write a meaningful commit message and push the commit. Note that this previous commit should not be left! See the concept of reseting the head and how to force push!

**Comment by Neel-Shah-29** [comment]
1) Change the naming convention of Course 3 to Course 4 since we have not done course 3 its course 4 on coursera.
2) I am not looking at the technical details of each assignment i assume you have checked it from expected output.

**Review by Neel-Shah-29 — APPROVED** [review]
LGTM!
If nothing yet to be added we can merge the PR.

File: Deep Learning/Course_1_Lakshaya.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
+# Neural Networks
+
+[Andrew NG course 1](https://youtube.com/playlist?list=PLkDaE6sCZn6Ec-XTbcX1uRg2_u4xOEky0 "Andrew NG course 1")
+
+Neural Networks use supervised learning where the ntwork needs to be trained using predefined, labeled data so the netwrok is tuned to make predictions. The performance of the network depends upon the complexity and scale of network and also the amount of data used to train it.
+
+![Performance vs Data](/Deep%20Learning/assets/Data%20vs%20Performance.png"Performance vs Data")
```
Comment (shahpratham) [inline Deep Learning/Course_1_Lakshaya.md]: Image is not rendered here

File: Deep Learning/Course_1_Lakshaya.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+ 
+![Ouput Matrix](/Deep%20Learning/assets/output%20matrix.png)
+
+## Logistic Regression
+
+The output tells us the probability of a certain case. Therefore, the out y^ lies between 0 and 1.
+
+We can use logistic regression only but that doesn't necessarily give us values between 0 and 1. Hence, we also make use of the sigmoid function. It takes input of any real number and maps it between 0 and 1.
+
+![Sigmoid Function](/Deep%20Learning/assets/Sigmoid%20function.png"Sigmoid Function")
```
Comment (shahpratham) [inline Deep Learning/Course_1_Lakshaya.md]: here as well

File: Deep Learning/assets/Course_1_Lakshaya.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
+# Neural Networks
```
Comment (shahpratham) [inline Deep Learning/assets/Course_1_Lakshaya.md]: remove this from assets

**Review by shahpratham — CHANGES_REQUESTED** [review]
@Greyless added few comments, otherwise LGTM.

**Review by shahpratham — APPROVED** [review]

**Pull request merged**